src/test/java/com/training/basetest/WebBaseTest.java

```diff
@@ -19,28 +25,36 @@
     public SoftAssertions softAssertions;
     private RemoteWebDriver driver;
     String execution;
+    public JsonObject jsonObject;
 
     @BeforeEach
     public void setup() throws MalformedURLException, NullPointerException {
+
+        File jsonFile = new File("src/test/java/resources/testData.json");
+        try {
+            JsonElement fileElement = JsonParser.parseReader(new FileReader(jsonFile));
+            jsonObject = fileElement.getAsJsonObject();
+        } catch (FileNotFoundException e) {
+            e.printStackTrace();
+        }
+
```

Comment:
Lets try to make this more modular by creating a `JsonReaderUtility` class in our `mainjava/com/training/utilities` package. and then create a function - `getJson(String key)` -> which would do the work of returning the value for the key passed. You can then use that function here and in all your tests